playbooks/configure/configure

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+read -e -p "Path for VMware VDDK file: " vddk_path
```

Comment:
Just wondering.
Why do you prefer `read` from a shell script over the [Ansible prompt module](https://docs.ansible.com/ansible/latest/user_guide/playbooks_prompts.html)?

Reply:
I wanted to run ansible-playbook from a shell script so that I can prefix the command with ANSIBLE_STDOUT_CALLBACK=debug

Reply:
Fair point. Just tested with Ansible 2.9 and the `vars_prompt` doesn't provide auto-complete.